### PR TITLE
feat: controlbar expand and drag hover and active states

### PR DESCRIPTION
**Pull request by cooper-joe**
This PR improves the user experience of the dashboards, fixing [UX-52](https://jira.dhis2.org/browse/UX-52), controlbar by:
- adding a `hover` and `active` state to the _Show more_ button, highlighting the fact it is 100% width.
- adding a `hover` and `active` state to the _Drag handle_ area.
- prolonging the time before the _show more_ tooltip is shown, preventing ui flickering.

These changes help the user understand which action will be triggered when clicking this action-dense area of the controlbar.

**Before:**

controlbar-before

**After:**

controlbar-after

### CODE REVIEW

**Review by jenniferarnesen — APPROVED** [review]

**Comment by cooper-joe** [comment]
Waiting for PM approval before merging.

**Review by janhenrikoverland — APPROVED** [review]

**Pull request merged**
